src/configureStore.js

```diff
@@ -1,18 +1,21 @@
-import { createStore, applyMiddleware } from 'redux'
+import { createStore, applyMiddleware, compose } from 'redux'
 import thunk from 'redux-thunk'
 import logger from 'redux-logger'
 import invoicesApp from './reducers'
 
 const configureStore = () => {
-  const middlewares = [ thunk ]
+  const middlewares = [thunk]
 
   if (process.env.NODE_ENV !== 'production') {
     middlewares.push(logger)
   }
 
+  const composeEnhancers =
+    window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose
+
   return createStore(
     invoicesApp,
-    applyMiddleware(...middlewares)
+    composeEnhancers(applyMiddleware(...middlewares))
```

Comment:
It is required for the "redux dev tools browser extension". https://github.com/zalmoxisus/redux-devtools-extension